tests/Makefile

```diff
@@ -18,8 +18,6 @@
 # If this block fails to do it automatically, replace with manual
 # setting: ARCH=x86 or ARCH=arm as appropriate
 
-.PHONY: clean ctCheck
```

Comment:
This should probably remain. It tells `make` that these targets need to be run whenever they are requested and must never be considered "fresh". Otherwise it is possible for `make` to think that the tests have already run, not become stale, and thus not re-check the output.

Reply:
OK, thanks: now the .PHONY target should be restored.